stock_custom_exchange_rate_date/tests/test_stock.py

```diff
@@ -80,7 +80,7 @@ def setUpClass(cls):
         cls.product = cls.env["product.product"].create(
             {
                 "name": "Product fifo",
-                "type": "product",
+                "type": "consu",
                 "default_code": "PR-FIFO",
```

Comment:
"consu" is the default product type. I think you should use `is_storable` instead, which is the new way to mark product to be inventoried.

Reply:
Done